datasets/sentinel-5p/collection/template.json

```diff
@@ -0,0 +1,238 @@
+{
+    "stac_version": "1.0.0",
+    "type": "Collection",
+    "id": "sentinel-5p-l2-netcdf",
+    "title": "Sentinel-5P Level-2",
+    "description": "{{ collection.description }}",
+    "license": "proprietary",
+    "links": [
+        {
+            "rel": "license",
+            "href": "https://sentinel.esa.int/documents/247904/690755/Sentinel_Data_Legal_Notice",
+            "type": "application/pdf",
+            "title": "Sentinel Data License"
+        },
+        {
+            "rel": "about",
+            "href": "https://sentinel.esa.int/web/sentinel/missions/sentinel-5p",
+            "type": "text/html",
+            "title": "Sentinel-5 Precursor Mission"
+        }
+    ],
+    "stac_extensions": [
+        "https://stac-extensions.github.io/sat/v1.0.0/schema.json",
```

Comment:
Is this extension necessary? I only see it under `summaries` (I'm not sure if that makes it necessary, but we don't have `eo` for https://planetarycomputer.microsoft.com/api/stac/v1/collections/sentinel-2-l2a).

Reply:
The example at https://github.com/radiantearth/stac-spec/blob/fd8fb4e3cf0b6c4a0fe74822accbc1725a92f92a/examples/collection.json does put `eo` in the collection `stac_extension`, despite only using it in summaries. So maybe we need to update our sentinel-2-l2a collection.

Reply:
>Is this extension necessary? I only see it under summaries 

My initial thought was, no, it is not necessary since the extension schema does not touch Collection summaries. But the [extension spec](https://github.com/radiantearth/stac-spec/tree/master/extensions) states:

>If a Collection [summary](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#summaries) contains Item fields that implement an extension, then the stac_extensions array of that Collection should list the extension ID.

So, per spec, it should be included. Which aligns with the example Collection you linked to.